frontend/src/components/SpecialistsList/SpecialistsList.tsx

```diff
@@ -64,40 +75,52 @@ export const SpecialistsList: FunctionComponent<Props> = props => {
-        text: 'any'
+        value: '',
+        text: 'Any'
     });
 
     return (
         <>
-            <Select
-                onChange={(event: React.SyntheticEvent<HTMLElement>, data: DropdownProps) =>
-                    locationChangeHandler(data)
-                }
-                options={locationsOptions}
-                placeholder="Select location"
-            />
-            {/*<Select*/}
-            {/*        options={paidOptions}*/}
-            {/*        onChange={(event: React.SyntheticEvent<HTMLElement>, data: DropdownProps) => paidChangeHandler(data)}*/}
-            {/*        placeholder="Paid"*/}
-            {/*/>*/}
-            <List divided relaxed>
-                {offeredServices.map(offeredService => (
-                    <List.Item key={offeredService.specialist.id}>
-                        <List.Header>{offeredService.specialist.name}</List.Header>
-                        <List.Content>
-                            <p>Price: {offeredService.price}</p>
-                            <p>Is prepaid: {offeredService.isPrepaid ? 'yes' : 'no'}</p>
-                            {user !== null ? (
-                                <Button color="green" onClick={() => requestServiceHandler(offeredService.id)}>
-                                    Request
-                                </Button>
-                            ) : null}
-                        </List.Content>
-                    </List.Item>
-                ))}
-            </List>
+            <Form className={styles.ServiceFilters} size="large">
+                <Form.Select
+                    onChange={(event: React.SyntheticEvent<HTMLElement>, data: DropdownProps) =>
+                        locationChangeHandler(data)
+                    }
+                    value={locationId}
+                    options={locationsOptions}
+                    placeholder="Select location"
+                />
+                {/*<Select*/}
+                {/*        options={paidOptions}*/}
+                {/*        onChange={(event: React.SyntheticEvent<HTMLElement>, data: DropdownProps) => paidChangeHandler(data)}*/}
+                {/*        placeholder="Paid"*/}
+                {/*/>*/}
+            </Form>
+            {offeredServices.length !== 0 ? (
+                <CardList>
+                    {offeredServices.map(offeredService => (
+                        <SpecialistCard
+                            key={offeredService.id}
+                            offeredService={offeredService}
+                            onServiceRequest={requestServiceHandler}
+                        />
+                    ))}
+                </CardList>
+            ) : (
+                <Segment className={styles.NoServiceSegment} placeholder>
+                    <Header icon>
+                        <Icon name="search" />
+                        {locationId
+                            ? "We don't have any specialists matching your filters."
```

Comment:
You should be consistent in whether you use single quotes or double quotes. (see also line below)
Do not you have a linter with a setting for the quotes ?

Reply:
Yes, I use single quotes in js code and double quotes in jsx code